### PR TITLE
Generate config files in volume (fix issue)

### PR DESCRIPTION
**Pull request by k-ayache**

### CODE REVIEW

**Comment by nlewo** [comment]
I think you can create this file at build time, in `dockerTools.buildImage.extraCommands` for instance.

File: pkgs/docker-images/default.nix [hunk 1]
```diff
@@ -23,7 +23,7 @@ let
     lib.buildImageWithPerps {
       inherit name services;
       fromImage = lib.images.kubernetesBaseImage;
-      extraCommands = "chmod u+w etc; mkdir -p var/log/contrail etc/contrail";
+      extraCommands = "chmod u+w etc; mkdir -p var/log/contrail etc/contrail; ln -s /run/consul-template-wrapper/contrail/vnc_api_lib.ini  etc/contrail/vnc_api_lib.ini";
```
Comment (eonpatapon) [inline pkgs/docker-images/default.nix]: Line is too long

Can you use a multiline string with '' ?

File: pkgs/docker-images/default.nix [hunk 2]
```diff
@@ -23,7 +23,9 @@ let
     lib.buildImageWithPerps {
       inherit name services;
       fromImage = lib.images.kubernetesBaseImage;
-      extraCommands = "chmod u+w etc; mkdir -p var/log/contrail etc/contrail";
+      extraCommands = "chmod u+w etc; \\
+                       mkdir -p var/log/contrail etc/contrail; \\
+                       ln -s /run/consul-template-wrapper/contrail/vnc_api_lib.ini  etc/contrail/vnc_api_lib.ini";
```
Comment (eonpatapon) [inline pkgs/docker-images/default.nix]: I meant using double simple quotes:

```
extraCommands = ''
  chmod u+w etc
  mkdir -p var/log/contrail etc/contrail
  ln -s /run/consul-template-wrapper/contrail/vnc_api_lib.ini  etc/contrail/vnc_api_lib.ini
''
```

**Comment by eonpatapon** [comment]
http://hydra.int0.aub.cloudwatt.net/eval/690

**Pull request merged**